pkg/agent/openflow/network_policy.go

```diff
@@ -1397,15 +1397,15 @@ func (c *client) ReassignFlowPriorities(updates map[uint16]uint16, table binding
 
 func parseDropFlow(flow string) (uint32, types.RuleMetric) {
 	// example format:
-	// table=101, n_packets=9, n_bytes=666, priority=200,ip,reg0=0x100000/0x100000,reg3=0x5 actions=drop
+	// table=101, n_packets=9, n_bytes=666, priority=200, reg0=0x100000/0x100000, reg3=0x5 actions=drop
```

Comment:
As long as we use slice to store I'm afraid that we still may meet similar issues. I suggest using a map instead of a slice to store segments, and we can use a unit test to guarantee the stat works as expected.

Reply:
Good point, please check the code I refactored for parsing flow